scss/ui/_flex.scss

```diff
@@ -69,4 +69,4 @@ $flex-grow-all: 2;
   .align-stretch {
     align-items: stretch;
   }     
-}
\ No newline at end of file
+}
```

Comment:
is this change necessary?

Reply:
@daRasmussen I guess it was a new line at the end issue together with margin change it shows that row 72 was change but it was actually a new row added on row 73 at first commit. 
So to answer your question: yes and no `:)`